src/components/ActiveVideo/ConnectionError/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { h, FunctionComponent } from 'preact'
+import { Button } from '../Button'
+import { Footer } from '../Footer'
+import { Header } from '../Header'
+import { ErrorIcon } from '../assets/ErrorIcon'
+import { Wrapper } from '../Wrapper'
+import { BaseScreen } from '../BaseScreen'
+import { localised } from '~locales'
+import { trackComponent } from 'Tracker'
+import type { WithLocalisedProps, WithTrackingProps } from '~types/hocs'
+import { StepComponentProps } from '~types/routers'
+
+type Props = StepComponentProps &
+  WithTrackingProps &
+  WithLocalisedProps & {
+    restart: () => void
+    retry: () => void
+  }
+
+const ConnectionError: FunctionComponent<Props> = ({
+  restart,
+  retry,
+  trackScreen,
+  translate,
+}: Props) => {
+  const handleRestart = (): void => {
+    trackScreen('connection_error_restart_clicked')
```

Comment:
Is this listed in the src/Tracker/trackData.tsx file? Otherwise it won't be send.

Analytics events also need to be added to ANALYTICS_CHANGELOG.md and the wiki: https://wiki.onfido.net/display/SI/SDK+Analytics+-+List+of+Events+and+Properties

Reply:
This was mapped on a previous [PR](https://github.com/onfido/onfido-sdk-ui/pull/2006/files#diff-cbfb202361ef05f48db4edd7b3d0e084221a6e640ee48cb551037f9c5062fbf9R1595) and added to the wiki. Tested it now and it seems like it's being sent correctly.